arch/lkl/kernel/threads.c

```diff
@@ -220,7 +220,7 @@ void threads_init(void)
 
 void threads_cnt_dec(void)
```

Comment:
No such one function is defined and used in the latest code base (a063e1631db5).